.github/workflows/github-actions.yml

```diff
@@ -8,43 +8,17 @@ on:
     branches: master
 
 jobs:
-  linux:
-    runs-on: ubuntu-latest
+  test:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu-latest, macos-latest]
     steps:
       - name: Checkout
         uses: actions/checkout@v3
       - name: Get package name
         id: cargo-name
-        uses: nicolaiunrein/cargo-get@master
+        uses: nicolaiunrein/cargo-get@1fc79e967ed81606e36eddeef51f870a873d5f4f
```

Comment:
Doesn't this now always test the same release or require a manual update?

Reply:
Unfortunately, there is no clean way to make this "dynamic" and remove the manual effort. GitHub Action syntax disallows us to use something like this: https://github.com/nicolaiunrein/cargo-get/actions/runs/5816914611/workflow

That's why pinning to master for this `self-test-action` isn't a bad idea knowing that you want to be informed of the error before the user of this action does.

Reply:
And of course this SHA is only temporary until I can see it working, but the final draft will be the same `master` again.